sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOrcScan.scala

```diff
@@ -827,7 +827,13 @@ trait OrcCommonFunctions extends OrcCodecWritingHelper { self: FilePartitionRead
 
     val table = withResource(new NvtxWithMetrics("ORC decode", NvtxColor.DARK_GREEN,
         metrics(GPU_DECODE_TIME))) { _ =>
-      Table.readORC(parseOpts, hostBuf, 0, bufSize)
+      try {
+        Table.readORC(parseOpts, hostBuf, 0, bufSize)
+      } catch {
+        case e: Exception => 
+          throw new IOException(s"""Error when processing file splits  
+                                 [${splits.mkString("; ")}]: $e""", e)
```

Comment:
We don't want triple quotes here, as that will encode the newline and all the extra spacing in the message string.
```suggestion
          throw new IOException("Error when processing file splits  " +
              s"[${splits.mkString("; ")}]", e)
```
Also applies to all the other places triple-quotes are used.